modules/nodes-descheduler-profiles.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-profiles_{context}"]
+= Descheduler profiles
+
+The following descheduler profiles are available:
+
+`AffinityAndTaints`:: This profile evicts pods in an effort to evenly spread similar pods, or pods of the same topology domain, among nodes.
++
+It enables the following strategies:
++
+* `RemovePodsViolatingInterPodAntiAffinity`: ensures that pods violating inter-pod anti-affinity are removed from nodes.
+* `RemovePodsViolatingNodeAffinity`: ensures that pods violating node affinity are removed from nodes.
+* `RemovePodsViolatingNodeTaints`: ensures that pods violating `NoSchedule` taints on nodes are removed.
++
+Pods with a node affinity type of `requiredDuringSchedulingIgnoredDuringExecution` are removed.
+
+`TopologyAndDuplicates`:: This profile evicts pods that violate basic pod and node affinity and node taints.
++
+It enables the following strategies:
++
+* `RemovePodsViolatingTopologySpreadConstraint`: finds unbalanced topology domains and tries to evict pods from larger ones when `DoNotSchedule` constraints are violated.
+* `RemoveDuplicates`: ensures that there is only one pod associated with a replica set, replication controller, deployment, or job running on same node. If there are more, then those duplicate pods are evicted for better spreading of pods in a cluster.
+
+`LifecycleAndUtilization`:: This profile evicts long-running pods and balances resource usage between nodes.
++
+It enables the following strategies:
++
+* `RemovePodsHavingTooManyRestarts`: ensures that pods that have been restarted too many times are removed from nodes.
++
+Pods that are restarted more than 100 times are removed. This includes restarts for Init Containers.
+
+* `LowNodeUtilization`: finds nodes that are underutilized and evicts pods, if possible, from overutilized nodes in the hope that recreation of evicted pods will be scheduled on these underutilized nodes.
++
+A node is considered underutilized if its CPU usage is less than 20%, its memory usage is less than 20%, and its number of pods is less than 20% of the allocated values.
++
+A node is considered overutilized if its CPU usage is over 50%, its memory usage is over 50%, or its number of pods is more than 50% of allocated values.
+
```

Comment:
If a node’s usage is above the configured target thresholds for any of the parameters, then the node’s pods might be considered for eviction. @bergerhoffer